lookout/style/format/analyzer.py

```diff
@@ -418,11 +418,20 @@ def _generate_token_fixes(
         code_generator = CodeGenerator(fe, skip_errors=True)
         new_vnodes = code_generator.apply_predicted_y(vnodes, vnodes_y, rule_winners, new_rules)
         token_fixes = []
+        newline_indx = CLASS_INDEX[CLS_NEWLINE]
         for line_number, line in self._group_line_nodes(
                 y, y_pred, vnodes_y, new_vnodes, rule_winners):
             line_ys, line_ys_pred, line_vnodes_y, new_line_vnodes, line_winners = line
-            new_code_line = code_generator.generate(
-                new_line_vnodes, "local").lstrip("\n").splitlines()[0]
+            new_code_line = code_generator.generate_new_line(new_line_vnodes)
+            if (new_line_vnodes and hasattr(new_line_vnodes[0], "y_old") and newline_indx in
+                    new_line_vnodes[0].y_old):
+                lines_num_diff = new_line_vnodes[0].y.count(newline_indx) - \
+                                 new_line_vnodes[0].y_old.count(newline_indx)
+                if lines_num_diff < 0:
+                    # Some line were removed. That means that several original lines should be
+                    # modified. GitHub suggestion feature cannot handle such cases right now.
+                    # To not confuse the user we do not provide any suggestion.
```

Comment:
One example:
Our code looks like 
```
codeline1\n
codeline2\n
codeline3\n
```
and style-analyzer removes first `\n`
so we have 
```
codeline1codeline2\n
codeline3\n
```
And it is not possible to just delete the second line. So all we can do is to suggest next change:
```
codeline1codeline2\n
codeline2\n
codeline3\n
```
or in two suggestions
```
codeline1codeline2\n
\n
codeline3\n
```

But both are bad and confusing.